types/tools/tool-settings.d.ts

```diff
@@ -45,5 +45,5 @@ export interface ToolSettings {
   /**
    * Tool's Toolbox settings
```

Comment:
Can you describe in jsdoc, what `false` means

Reply:
Fixed.